msteams-platform/messaging-extensions/api-based-overview.md

```diff
@@ -21,6 +21,8 @@ Message extensions built using API (API-based) use a web service to manage user
 * Retrieve knowledge-based information, for example, my team’s design files in Figma.
 * Perform actions on behalf of the user, for example, create a Contoso ticket.
 
+You can implement Microsoft Entra ID Single Sign-On and API key-based authentication in API-based search message extensions to provide secure and seamless access to applications. Secure authentication methods are critical for creating robust and user-friendly applications. To enable these authentication methods, you can update your app manifest with the `none`, `oauth`, `apiSecretServiceAuth`, `microsoftEntra`, or `userAuth` authentication methods.
```

Comment:
We support all these values? The start of the paragraph mentions only entra and api key.

Reply:
oauth is not supported. Currently the supported properties are `none`, `apiSecretServiceAuth`, and `microsoftEntra.